submissions/5mountains/popup/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>pure html/css Popup</title>
+    <link rel="icon" href="./img/favicon.png" type="image/png">
+    <link rel="stylesheet" href="./css/main.css">
+</head>
+<body>
+    <nav class="nav">
+        <ul class="nav__menu">
+            <li class="menu__item"><a class="menu__link-text" href="https://mail.google.com/mail/?tab=wm&authuser=0&ogbl">Mail</a></li>
+            <li class="menu__item"><a class="menu__link-text" href="https://www.google.com.ua/imghp?hl=ru&tab=wi&authuser=0&ogbl">Images</a></li>
+            <li class="menu__item-button">
+                <input type="checkbox" id="menu__input" class="menu__item-input">
+                <label for="menu__input" class="menu__item-label" tabindex="0"></label>
+                <ul class="menu__submenu">
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/calendar.png" alt="calendar icon">
+                            Calendar
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#" alt="mail icon">
```

Comment:
It seems that you wanted to add the `alt` attribute to the `img` tag but accidentally added to `a` tag.
Please, fix it

Reply:
I think I was rushing copy-past or so) and I did so stupid mistakes)

Reply:
No problem! You did an awesome job! Just try not to rush and check your code several times 🙂